### ISSUE TITLE
Change one list of mouse-action letters to two separate lists (with second one being optional)

### ISSUE THREAD

**Issue by vlnn** [croian/mouseless-issues#38]
### Problem Description

Current pair generation algorithm creates rather hard-to-type pair; some users may prefer either to create pairs that are easy to roll (e.g. `qw`, `we`, `er` but not `qe` or `qc`), or vice versa restrict pair to be on different parts of the keyboard (so first letter is always on left side of the split keyboard, and the second one is on the right side).

### Proposed Solution

I propose add optional "secondary" mouse action keys field. When it's empty, behavior is the same as right now. When it has letters in it, first letter of the pair is always being taken from first action keys field, the second one is always being taken from the secondary mouse action keys field.

**Comment by vlnn** [comment]
yes, this makes sense — and BTW right now it is possible to put several of same characters in the mouse action keys, which creates "never-clicked" cells.

**Comment by croian** [comment]
This is now done!  Pending release

![Image](https://github.com/user-attachments/assets/81eee26f-3498-4364-916d-86068338f0b5)

**Comment by vlnn** [comment]
Oh that's something I'm waiting so bad! Thanks @croian

**Comment by croian** [comment]
in [v0.4.0-preview.0](https://github.com/croian/mouseless/releases/tag/v0.4.0-preview.0)
